src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -55,33 +77,54 @@
 helps['account show'] = """
 type: command
 short-summary: Get the details of a subscription.
-long-summary: If no subscription is specified, shows the current subscription.
+long-summary: >
+    If no subscription is specified, information for the current subscription is returned.
+examples:
+    - name: >
+        Get the current default subscription returning results as JSON.  Use `--output yaml` for plain text. 
```

Comment:
JSON is the default, but may be overridden by `defaults.output` config option. `yaml` is for YAML, not plain text.

Reply:
You are making an assumption that the customer hasn't changed their defaults, but I changed it to `--output table` as the majority of customers will have kept system defaults as you assumed.